benchmarking/commons/launch_remote_sagemaker.py

```diff
@@ -30,9 +31,11 @@
 from benchmarking.commons.hpo_main_sagemaker import parse_args
 from benchmarking.commons.utils import (
     message_sync_from_s3,
-    basic_cpu_instance_sagemaker_estimator,
     find_or_create_requirements_txt,
 )
+from syne_tune.backend.sagemaker_backend.estimators import (
```

Comment:
It is a bit funny to have this in `sagemaker_backend`, because it is used for remote launching.

Reply:
Ok makes sense, moved it to remote so we can disentangle the backend code and the plumbing code